Automattic/wp-calypso
# Checkout: Fix Payment Box Radio

#### Changes proposed in this Pull Request

* Updates some z-indexes so that it's possible to click the payment box radio

#### Testing instructions



* Apply radios test of the A/B test added in #36705 
* Visit the Checkout 
* Verify you can click radios

![ezgif-6-f93629899d72](https://user-images.githubusercontent.com/43215253/67856742-2119db00-fb0d-11e9-9edf-e1ecd770ed14.gif)

cc @fditrapani, @nbloomf 

Fixes #37175
